apps/fz_http/lib/fz_http_web/auth/json/authentication.ex

```diff
@@ -0,0 +1,20 @@
+defmodule FzHttpWeb.Auth.JSON.Authentication do
+  @moduledoc """
+  API Authentication implementation module for Guardian.
+  """
+  use Guardian, otp_app: :fz_http
+  alias FzHttp.Users
+
+  @impl Guardian
+  def subject_for_token(resource, _claims) do
+    {:ok, to_string(resource.id)}
+  end
+
+  @impl Guardian
+  def resource_from_claims(%{"sub" => id}) do
+    case Users.get_user(id) do
```

Comment:
Should we check `user.disabled_at` here too? After the JWT token is issued you can't revoke it so we kinda want to make sure that the user that made the request still has permission to make it even if he was authorized before deactivation. (Or maybe you do it already somewhere else and I didn't find the place?)

Reply:
Good catch! The `disabled_at` currently only applies to users with `user.role == :unprivileged` for their VPN connection, and is not used to control auth at this time.

We actually plan to [implement token revocation](https://github.com/firezone/firezone/blob/feat/rest-api/apps/fz_http/lib/fz_http/api_tokens/api_token.ex#L14) (I know, I know, "defeats the purpose of JWT", but we'll be using a cached blacklist of `api_token` ids to check this quickly with an ETS table).

The API tokens will be creatable only by admins and could be fairly long-lived (90 days) so we do need some way to revoke them. Perhaps using a JWT is overkill but decided to use it since we already have Guardian set up.